src/tools/std20/iterator/operations/test/prev.test.cpp

```diff
@@ -0,0 +1,78 @@
+// include Catch2
+#include <catch2/catch_test_macros.hpp>
+
+// what we are testing
+#include "tools/std20/iterator.hpp"
+
+// other includes
+#include <list>
+#include <vector>
+
+// convenience typedefs
+//using namespace njoy::tools;
+namespace std20 = nano;
+
+SCENARIO( "prev" ) {
+
+  WHEN( "when using a container with bidirectional iterators" ) {
+
+    std::list< int > values = { 1, 2, 3, 4, 5 };
+
+    THEN( "advance can be used" ) {
```

Comment:
"advance" -> "prev"

Likewise with other THENs in this scenario

Reply:
same